crane-cloud/backend
# Improved Email templates

# Description

- This PR aims to improve the anonymous_user_to_project email using the standard emailing template created in Figma. It takes care of the standard branding, sizes, font styles, general layout, and background formats.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

1. https://trello.com/c/IWyB6AVy  
2. https://trello.com/c/IQqVNWgZ

## How Can This Be Tested?

- Run the branch on your local machine. 
- Send an invitation email to an anonymous user.
- Check its appearance on various devices for font size, colors, etc.
- Test the social media icons for Medium, LinkedIn, YouTube, and X to ensure they link to the official Crane Cloud social media platforms.

## Screenshot
![Crane Cloud](https://github.com/crane-cloud/backend/assets/87686674/203e8c17-f6f5-4a1d-97e6-a96cebca3170)

## Checklist:
